inc/MicroBitRadio.h

```diff
@@ -76,6 +76,13 @@ namespace codal
 #define MICROBIT_RADIO_MAXIMUM_RX_BUFFERS       4
 #define MICROBIT_RADIO_POWER_LEVELS             8
 
+// Max packet size is configurable, so ensure maximum value is not exceeded
+// TODO: Update this value once issue codal-microbit-v2#383 is resolved
+// https://github.com/lancaster-university/codal-microbit-v2/issues/383
+#if MICROBIT_RADIO_MAX_PACKET_SIZE > 250
+    #error "MICROBIT_RADIO_MAX_PACKET_SIZE cannot be larger than 250 bytes"
+#endif
```

Comment:
This can probably be increased to 252, as 3 bytes are used as part of the DAL/CODAL header, but I'm not 100% sure, so used  250 as a safe value.

When issue https://github.com/lancaster-university/codal-microbit-v2/issues/383 is fully resolved this should be updated.